plugin/code_actions.py

```diff
@@ -308,10 +309,14 @@ def combine_commands(self) -> 'List[Tuple[str, str, CodeActionOrCommand]]':
 
     def show_code_actions(self) -> None:
         if len(self.commands) > 0:
-            items = [command[1] for command in self.commands]
             window = self.view.window()
             if window:
-                window.show_quick_panel(items, self.handle_select, placeholder="Code action")
+                items, selected_index = format_code_actions_for_quick_panel([command[2] for command in self.commands])
```

Comment:
Seems like the `self.commands` tuple doesn't need the middle (`title`) element now.